src/i18n/en.js

```diff
@@ -57,6 +58,7 @@ export default {
   forgot_password: {
     back_login: `Did you remember your Password? Go back to Log in.`,
     back_login_after_forgot: `Back to Log in`,
+    blocked_account: `Your account has been cancelled. To know more please <a href="${urlSiteContact}">contact us</a>`,
```

Comment:
Remember to add the suffix HTML to avoid confusions using this value.

Also, it should use the target blank, right?

And should a dot finish the sentence?

```suggestion
    blocked_account_HTML: `Your account has been cancelled. To know more please <a href="${urlSiteContact}" target="_blank">contact us</a>.`,
```

Another option is using Markdown, I think that it should be our preferred option:

```suggestion
    blocked_account_MD: `Your account has been cancelled. To know more please [contact us](${urlSiteContact}).`,
```

If you choose Markdown, you need to change the usage of this value in _Login.js_, something like this:

```diff
    case 'BlockedAccount':
-      return { _error: <FormattedHTMLMessage id="forgot_password.blocked_account" /> };
+      return { _error: <FormattedMessageMarkdown id="forgot_password.blocked_account_MD" /> };
```

Reply:
subido